### PR TITLE
Basic Rust layout

### PR DESCRIPTION
**Pull request by Alecaddd**
## Summary / How this PR fixes the problem?
Implement a basic 3 pane layout in Rust

### CODE REVIEW

File: src/layout/left_sidebar.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+impl Default for LeftSidebar {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl LeftSidebar {
+    pub fn new() -> Self {
+        glib::Object::builder()
```
Comment (bilelmoussaoui) [inline src/layout/left_sidebar.rs:50]: This could be moved to the Default impl, you can drop the new method then

File: src/layout/mod.rs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * Akira is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * Akira. If not, see <https://www.gnu.org/ licenses/>.
+ */
+pub mod left_sidebar;
+
+pub mod sidebars {}
```
Comment (bilelmoussaoui) [inline src/layout/mod.rs:19]: Seems unused

File: src/window.rs [hunk 3]
```diff
@@ -132,6 +114,60 @@ impl AppWindow {
...
+
+        let gtk_settings = gtk::Settings::default().expect("Unable to get GtkSettings object");
+        mode_switch
+            .bind_property("active", &gtk_settings, "gtk-application-prefer-dark-theme")
+            .bidirectional()
+            .build();
+
+        let header_bar = gtk::HeaderBar::builder().show_title_buttons(true).build();
+
+        header_bar.style_context().add_class("default-decoration");
```
Comment (bilelmoussaoui) [inline src/window.rs:135]: Use add_css_class, this method is deprecated

**Review by bilelmoussaoui — APPROVED** [review]

File: src/layout/mod.rs (hunk 2 above)
Comment (Alecaddd) [reply src/layout/mod.rs:19]: Yes, thank you, I was experimenting with Rust modules and I need to clean this up

File: src/window.rs (hunk 3 above)
Comment (Alecaddd) [reply src/window.rs:135]: Nice!

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request merged**